andela/mrm_api
# #161778500 Ratio of cancelled meetings to bookings

#### What does this PR do?
- Adds the functionality to get the ratio of bookings to cancelled meetings and check ins for an individual room. It also adds on a percentage of cancellations and checkins

#### How should this be manually tested?
- Run the server 
- On `localhost:5000/mrm` run the `analyticsRatios` query with `startDate` and `endDate` as parameters
- This should return the total ratio for all the rooms put together. A screenshot example will be attached below
- On the same URL run the query `analyticsRatiosPerDay` with `startDate` and `endDate` as parameters
- This should return the bookings to cancellation/checkins ratio of each individual room. A screenshot example will be attached below

#### What are the relevant pivotal tracker stories?
[#161778500](https://www.pivotaltracker.com/story/show/161778500)

#### Screenshots 
**Query for sum of all room ratios**
<img width="985" alt="screenshot 2018-11-09 at 16 06 56" src="https://user-images.githubusercontent.com/33119403/48264596-1917e580-e43b-11e8-864e-7968ce7c6cae.png">

**Query for individual room ratios**
<img width="1149" alt="screenshot 2018-11-12 at 11 12 14" src="https://user-images.githubusercontent.com/33119403/48334520-00424680-e66c-11e8-8658-672598484f7b.png">
